main.py

```diff
@@ -34,56 +34,56 @@
     has_wandb = False
 
 
-def quantize_model(model, args):
+def quantize_model(model: PreTrainedModel, args: Namespace):
     """main entry point to functions for model quantization"""
     tick = time.time()
     print("Loading data ...")
-    dataloader = get_loaders(
+    data = get_loaders(
         args.dataset,
         nsamples=args.nsamples,
         seed=args.seed,
         model_path=args.model_path,
-        seqlen=model.seqlen,
+        seqlen=args.model_seqlen,
     )
     if args.val_size > 0:
-        all_ids = torch.randperm(len(dataloader))
+        all_ids = torch.randperm(len(data))
         train_ids, val_ids = all_ids[args.val_size :], all_ids[: args.val_size]
-        train_dataloader = [dataloader[i] for i in train_ids]
-        val_dataloader = [dataloader[i] for i in val_ids]
+        train_data = [data[i] for i in train_ids]
+        val_data = [data[i] for i in val_ids]
     else:
-        train_dataloader = dataloader
-        val_dataloader = None
+        train_data = data
+        val_data = None
 
-    results = quantize_aq(model, train_dataloader, args, val_dataloader)
+    results = quantize_aq(model, train_data, val_data, args)
     print(f"quantization time: {time.time() - tick:.1f}")
     return results
 
 
 @torch.no_grad()
 def get_inps(
-    model: PreTrainedModel, data_iterable: Iterable, args: Namespace, nsamples: Optional[int] = None
-) -> Sequence[torch.Tensor]:
+    model: PreTrainedModel,
+    data: Sequence,
+    model_seqlen: int,
+    devices: Sequence[torch.device],
+    offload_activations: bool,
+) -> Tuple[Sequence[torch.Tensor], Dict]:
```

Comment:
I tested that running perplexity_eval with old vs new get_inps returns exactly the same perplexity